.github/workflows/release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Compute release name and tag
         id: release_info
         run: |
-          if [[ $IS_NIGHTLY ]]; then
+          if [[ ${IS_NIGHTLY} == 'true' ]]; then
```

Comment:
nit: should this also not access by `${{ env.IS_NIGHTLY == 'true' }}`?

Reply:
I don't think that will work as bash won't understand the `env.` construct which is specific to gh actions